packages/app-form-builder/src/types.ts

```diff
@@ -11,6 +11,44 @@ import {
 import { ApolloClient } from "apollo-client";
 import { SecurityPermission } from "@webiny/app-security/types";
 
+export type DropTargetType = "field" | "row" | "conditionGroup" | "step";
```

Comment:
Does it make sense to use this instead of `DroppableUIElement` in `packages/app-form-builder/src/admin/components/FormEditor/Droppable.tsx:23` ?